build-farm/make-adopt-build-farm.sh

```diff
@@ -152,5 +152,8 @@ export BUILD_ARGS="${BUILD_ARGS} --use-jep319-certs"
 
 echo "$PLATFORM_SCRIPT_DIR/../makejdk-any-platform.sh" --clean-git-repo --jdk-boot-dir "${JDK_BOOT_DIR}" --configure-args "${CONFIGURE_ARGS_FOR_ANY_PLATFORM}" --target-file-name "${FILENAME}" ${TAG_OPTION} ${OPTIONS} ${BUILD_ARGS} ${VARIANT_ARG} "${JAVA_TO_BUILD}"
 
+# Convert all speech marks in config args to make them safe to pass in.
+CONFIGURE_ARGS_FOR_ANY_PLATFORM="${CONFIGURE_ARGS_FOR_ANY_PLATFORM//\"/temporary_speech_mark_placeholder}"
```

Comment:
@adamfarley OK I get the intent and your example is great - but I'm not grokking this source code.

Why do we need the long `temporary_speech_mark_placeholder` 'variable' here

Reply:
It seemed the easiest way to preserve the speech marks as they were passed into the makejdk-any-platform.sh script without interfering with the speech marks around the wider CONFIGURE_ARGS_FOR_ANY_PLATFORM parameter.

I experimented with escaping the escapes of my escapes, but it seemed a convoluted way that was easy to break. This way we can pass the variable around as much as we like, and the speech marks survive until we use them in build.sh

Reply:
OK, I get it now - can you add a comment to both lines telling the user where it's twin declaration is?

Reply:
Sure thing. Uno momento.

Reply:
Ok, meetings are over. Changes made.
I also updated the docs in a couple of places, to ensure people can use the new feature both pre-and-post repo split.